src/Symfony/Component/Mailer/Transport/Smtp/Stream/AbstractStream.php

```diff
@@ -37,7 +37,7 @@ public function write(string $bytes, bool $debug = true): void
     {
         if ($debug) {
             foreach (explode("\n", trim($bytes)) as $line) {
-                $this->debug .= sprintf("> %s\n", $line);
+                $this->debug .= sprintf("[%s] > %s\n", (new \DateTime())->format('c'), $line);
```

Comment:
```suggestion
                $this->debug .= sprintf("[%s] > %s\n", date('c'), $line);
```

Or better: move that to a variable before the loop.
Also, what about a shorter format?